<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplingWithFrameTrackWidget</class>
 <widget class="QWidget" name="SamplingWithFrameTrackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vertical_layout_">
   <item>
    <layout class="QHBoxLayout" name="input_layout_">
     <item>
      <widget class="orbit_mizar_widgets::SamplingWithFrameTrackInputWidget" name="baseline_input_" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>250</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="orbit_mizar_widgets::SamplingWithFrameTrackInputWidget" name="comparison_input_" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>250</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="multiplicity_correction_layout_">
     <item>
      <widget class="QCheckBox" name="multiplicity_correction_">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A &lt;span style=&quot; font-style:italic;&quot;&gt;false alarm &lt;/span&gt;is an event of reporting a statistically significant difference in the absence of such. In our case it is an event of the tool reporting that the runtime of a function significantly differs between the captures by mistake, not due to the presence of real difference. &lt;/p&gt;&lt;p&gt;When the multiplicity correction is &lt;span style=&quot; font-style:italic;&quot;&gt;off, &lt;/span&gt;the probability of a false alarm &lt;span style=&quot; font-style:italic;&quot;&gt;for each function&lt;/span&gt; is controlled by the drop down list to the right. &lt;/p&gt;&lt;p&gt;When the multiplicity correction is &lt;span style=&quot; font-style:italic;&quot;&gt;on&lt;/span&gt;, the probability of &lt;span style=&quot; font-style:italic;&quot;&gt;at least one&lt;/span&gt; false alarm is controlled by the same drop down list. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Multiplicity Correction</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="significance_level_label_">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="significance_level_">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentText">
        <string>0.05</string>
       </property>
       <item>
        <property name="text">
         <string>0.05</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.01</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="use_symbols_toggle_">
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentText">
        <string>Use Baseline symbols</string>
       </property>
       <item>
        <property name="text">
         <string>Use Baseline symbols</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Use Comparison symbols</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="update_button_">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="orbit_mizar_widgets::SamplingWithFrameTrackOutputWidget" name="output_" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>orbit_mizar_widgets::SamplingWithFrameTrackInputWidget</class>
   <extends>QWidget</extends>
   <header>MizarWidgets/SamplingWithFrameTrackInputWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>orbit_mizar_widgets::SamplingWithFrameTrackOutputWidget</class>
   <extends>QWidget</extends>
   <header>MizarWidgets/SamplingWithFrameTrackOutputWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
